src/unity/python/README.md

```diff
@@ -1,132 +0,0 @@
-<img align="right" src="https://docs-assets.developer.apple.com/turicreate/turi-dog.svg" alt="Turi Create" width="100">
```

Comment:
You can delete this file entirely.

Reply:
I did. GH has a very strange way of showing it in the diff here...
if you'll checkout my branch and do `git log --name-status` you can see it's deleted.